backend-sb/src/webplayer/webplayer.gateway.ts

```diff
@@ -0,0 +1,55 @@
+import {
+  WebSocketGateway,
+  WebSocketServer,
+  SubscribeMessage,
+  OnGatewayConnection,
+  OnGatewayDisconnect,
+  MessageBody,
+} from '@nestjs/websockets';
+import { Server } from 'socket.io';
+import { Logger } from '@nestjs/common';
+
+@WebSocketGateway()
+export class WebplayerGateway implements OnGatewayConnection, OnGatewayDisconnect {
+  private logger = new Logger('WebplayerGateway');
+
+  @WebSocketServer()
+  private readonly server: Server;
+  private users: number = 0;
+  private currentURI: string[] | undefined = undefined;
+
+  async handleConnection() {
+    // A client has connected
+    this.users++;
+
+    this.logger.verbose(`Current number of users: ${this.users}`);
+
+    // Notify connected clients of current users
```

Comment:
Where's the code for this comment? The same comment below had a line of code. I'm wondering whether that's missing.

Reply:
It got removed. I had an event called `notifyUsers` but I didn't have time to implement a notification system on the frontend. It got scraped, we'll do it in the next PR. Will remove the comment.